src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
+        setArea(side * side);
+    }
```

Comment:
Method `obtainTheArea` should be private if it is only intended to be used within this class. Moreover, it's not recommended to have a method that is called only once and from the constructor. You could directly calculate the area in the constructor or in the overridden `getArea()` method if it's meant to be a part of the class's interface.